src/components/common/styles.js

```diff
@@ -99,7 +119,7 @@ export const StyledDot = styled.div`
   border-radius: 1rem;
 `;
 
-/* login page styled components */
```

Comment:
If these styled components are being only used for Landing.js, I'd move them there. If a component is used in multiple files, the it can go in styles.js. I just don't want to bloat this file up with too many things.

Reply:
Sounds good, I'll remove any styled components which are not used across more than one file from styles.js

Reply:
Most of the styled components I used are used across Landing.js, Login.js, and Signup.js